Models/c172p.xml

```diff
@@ -1020,6 +1020,15 @@
         </offsets>
     </model>
     <model>
+        <name>ControlLock</name>
+        <path>Interior/Control-lock/Control-lock.xml</path>
```

Comment:
When you change the capitalization of the folder, please remember to change here as well